Makefile.PL

```diff
@@ -28,15 +16,11 @@ WriteMakefile(
 	'Slay::Maker' => 0.04,
 	'File::Path'  => 0,
 	'File::Copy::Recursive' => 0,
+        'ExtUtils::MakeMaker' => 7.66,
```

Comment:
Why require such a recent MakeMaker?

Reply:
Because the makfile does a bunch of contortions to deal with older makefiles, a simpler solution is to just require a modern one.

Reply:
That doesn't explain why it would need more than the `6.54` that was previously suggested